.gitignore

```diff
@@ -10,3 +11,4 @@ pom.xml.asc
 /.nrepl-port
 /.cpcache
 /.clj-kondo
+/.lsp
```

Comment:
This change should be removed, as they're not relevant to the PR.

Reply:
I think this was a VSCode-generated directory that I didn't want to accidentally commit. I can remove it.